src/mlpack/tests/main_tests/pca_test.cpp

```diff
@@ -96,23 +93,21 @@ BOOST_AUTO_TEST_CASE(PCANoVarRetainTest)
   mlpackMain();
 
   // Check that the output has 1 dimensions.
-  BOOST_REQUIRE_EQUAL(IO::GetParam<arma::mat>("output").n_rows, 1);
-  BOOST_REQUIRE_EQUAL(IO::GetParam<arma::mat>("output").n_cols, 5);
+  REQUIRE(IO::GetParam<arma::mat>("output").n_rows == 1);
+  REQUIRE(IO::GetParam<arma::mat>("output").n_cols == 5);
 }
 
 /**
  * Check that we can't specify an invalid new dimensionality.
  */
-BOOST_AUTO_TEST_CASE(PCATooHighNewDimensionalityTest)
+TEST_CASE_METHOD(PCATestFixture, "PCATooHighNewDimensionalityTest", "[PCAMainTest]")
```

Comment:
Looks like this line is longer than 80 characters; would you mind wrapping it? :) I'm surprised the style checker didn't catch the issue.

There may be a few other lines where this is true---I didn't check them all (I'm using a phone right now so it's tedious :)).